CHANGELOG.rst

```diff
@@ -2,13 +2,16 @@
 CHANGELOG
 =========
 
+1.18.0
+======
+
+* feature: Support for Horovod
 
 1.17.3
 ======
 
 * bug-fix: Handle StopIteration in CloudWatch Logs retrieval
```

Comment:
Did we release 1.17.3? If not these should be under 1.18.0

Reply:
yeah I removed 1.17.3 from changelog